pdr_backend/pdr_dashboard/dash_components/plots.py

```diff
@@ -338,3 +341,95 @@ def get_figures_and_metrics(
     figs_metrics.make_figures()
 
     return figs_metrics
+
+
+@enforce_types
+def get_feed_figures(payouts: Optional[List], subscriptions: List):
+    """
+    Return figures for a selected feed from the feeds table.
+    """
+
+    # Initialize empty figures with default settings
+    sales_fig = create_figure(
+        [], "Sales", "Daily Sales (OCEAN)", False, ticker_format="%m-%d"
+    )
+    revenues_fig = create_figure(
+        [], "Revenues", "Daily Revenues (OCEAN)", False, ticker_format="%m-%d"
+    )
+    accuracies_fig = create_figure(
+        [], "Accuracy", "Avg Accuracy (OCEAN)", False, yaxis_range=[40, 70]
+    )
+    stakes_fig = create_figure([], "Stakes", "Stake (OCEAN)", False)
+    predictions_fig = create_figure([], "Predictions", "Predictions", False)
+    profits_fig = create_figure([], "Profits", "Pdr Profit (OCEAN)", False)
+
+    if not payouts or not subscriptions:
+        return (
+            sales_fig,
```

Comment:
we can use a NamedTuple or a class (or dataclass) here and also at the return statement. Long tuples are always confusing to maintain.